common/common.vcxproj

```diff
@@ -83,10 +88,17 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="EventSource.inl" />
+    <MASM Include="FastJmp.asm">
+      <FileType>Document</FileType>
+      <PreprocessorDefinitions Condition="'$(Platform)'=='Win32'">_M_X86_32;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions Condition="'$(Platform)'=='x64'">_M_X86_64;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions Condition="$(Configuration.Contains(AVX2))">_M_X86_64;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
I think this was supposed to be `_M_AVX2`?

Though I don't think you should have to `vzeroupper`, if the upper halves are volatile no one should be relying on them being any particular value, whether that's zero or whatever garbage the function that called `fastjmp_jmp` left there (probably also zero if the compiler is trying to avoid transition penalties).

Reply:
Whoops, indeed, copy/paste fail.

I was thinking of the rec frames more than C frames. But I suppose they don't use 256bit ops, so this is probably fine to omit?

Reply:
> But I suppose they don't use 256bit ops, so this is probably fine to omit?

Yeah not right now, x86emitter can't even emit them yet
Going to have to think about how we deal with it in the future, but there's a lot of other C functions we call from the recompiler that aren't fastjmp_jmp so we'll still need to deal with those either way